charts/cromwell/templates/_deployment.tpl

```diff
@@ -7,7 +7,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ $settings.name }}
+  name: {{ $settings.name }}-deployment
```

Comment:
We were going to get rid of this rename and an exception for Cromwell, right @knapii-developments?

Reply:
Oh I thought it was for the service accounts. 04b7e36